pkg/reconciler/buildrun/resources/sources/git.go

```diff
@@ -6,18 +6,25 @@ package sources
 
 import (
 	"fmt"
+	"strconv"
 	"strings"
+	"time"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	build "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
 	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
 	"github.com/shipwright-io/build/pkg/config"
+
 	pipelineapi "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
-	corev1 "k8s.io/api/core/v1"
 )
 
 const (
-	commitSHAResult    = "commit-sha"
-	commitAuthorResult = "commit-author"
-	branchName         = "branch-name"
+	commitSHAResult       = "commit-sha"
+	commitAuthorResult    = "commit-author"
+	commitTimestampResult = "commit-timestamp"
```

Comment:
Consider using a common result as suggested in bundle.go. This will help us when we want to consume the result from another step (image-processing). Ideally we there need no check which result to read but rely on a common name independent of the source type.